app/lib/Plugins/SearchEngine/ElasticSearch.php

```diff
@@ -239,7 +239,7 @@ private function addFragmentToUpdateContentBuffer(array $pa_fragment, array $pa_
 	protected function getClient() {
 		if(!self::$client) {
 			self::$client = Elasticsearch\ClientBuilder::create()
-				->setHosts([$this->elasticsearch_base_url])
+				->setHosts([parse_url($this->elasticsearch_base_url)])
```

Comment:
We've been using the AWS managed ElasticSearch with the 5.x version of the ElasticSearch plugin, and had to parse the url otherwise the request would go to `://awsclusterhostname.aws.amazon.com` rather than `https://...`.

Reply:
Got it. I just tested this out and it works fine.